pkgs/tools/misc/mpy-utils/default.nix

```diff
@@ -0,0 +1,20 @@
+{ lib, python3, buildPythonApplication, fetchPypi }:
+
+buildPythonApplication rec {
+  pname = "mpy-utils";
+  version = "0.1.13";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "sha256-die8hseaidhs9X7mfFvV8C8zn0uyw08gcHNqmjl+2Z4=";
+  };
+
+  propagatedBuildInputs = with python3.pkgs; [ fusepy pyserial ];
```

Comment:
if you use `python3Packages.callPackage` you can remove the `with` and get the packages as parameters